README.md

```diff
@@ -151,18 +230,19 @@ CREATE TABLE persons(
 
 Write Ruby code that will create an instance of a person.
 
-Your answer...
+
 
 ```ruby
-Replace this with your answer
+Person.create( { :name => "Vince", :age => "22"} )
```

Comment:
:+1: although curly brackets are optional in this example. Could also define attributes using this format - `name: "Vince"`